tests/test_pip_install_saved_bundle.py

```diff
@@ -18,12 +18,11 @@ def test_pip_install_saved_bentoservice_bundle(bento_bundle_path, tmpdir):
     ).decode('utf-8')
 
     assert "Processing {}".format(bento_bundle_path) in stdout
-    assert "Collecting bentoml=={}".format(get_bentoml_deploy_version()) in stdout
+    assert "Building wheels for collected packages: ExampleBentoService" in stdout
     assert "Successfully built ExampleBentoService" in stdout
 
-    # ensure BentoML is installed as dependency
-    assert os.path.isfile(os.path.join(install_path, "bin/bentoml"))
-    assert os.path.isdir(os.path.join(install_path, "bentoml"))
+    assert os.path.isfile(os.path.join(install_path, "bin/ExampleBentoService"))
```

Comment:
this part is testing `bentoml` is installed... not testing if ExampleBentoService is installed...

testing ExampleBentoService installed is covered separately down below